CHANGELOG.md

```diff
@@ -1,18 +1,72 @@
+  This field was deprecated in v0.116.0 and will be removed in v0.120.0.
+
+- (Core) `otelcol`: Remove warnings when 0.0.0.0 is used ([#11713](https://github.com/open-telemetry/opentelemetry-collector/issues/11713), [#8510](https://github.com/open-telemetry/opentelemetry-collector/issues/8510))
+
+
+### 🚩 Deprecations 🚩
+
+- (Contrib) `sapmexporter`: Deprecate SAPM exporter ([#36028](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36028))
+  The SAPM exporter is being marked as deprecated. Please use the `otlphttp` exporter with the configuration shown
+  below. Also update your pipeline configuration for Traces accordingly.
+  ```yaml
+    exporters:
+        otlphttp:
+            traces_endpoint: "${SPLUNK_INGEST_URL}/v2/trace/otlp"
+            headers:
+                "X-SF-Token": "${SPLUNK_ACCESS_TOKEN}"
+    ```
+
 
 ### 🚀 New components 🚀
 
 - (Splunk) Add `metricsgeneration` processor ([#5769](https://github.com/signalfx/splunk-otel-collector/pull/5769))
 
+
 ### 💡 Enhancements 💡
 
 - (Splunk) Add a new discovery bundle for Envoy proxy metrics ([#5780](https://github.com/signalfx/splunk-otel-collector/pull/5780))
+- (Contrib) `k8sclusterreceiver`: Add additional attributes to node and pod entities ([#35879](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35879))
+  Adds the following attributes to node and pod metadata/entities:
+    - `k8s.pod.phase`: The phase of a Pod indicates where the Pod is in its lifecycle. E.g. 'Pending', 'Running'
+    - `k8s.pod.status_reason`: A brief message indicating details about why the pod is in this state. E.g. 'Evicted'
+    - `k8s.node.condition_*`: The condition of a node. e.g. `k8s.node.condition_ready`. The value can be `true`, `false`, `unknown`.
+
+- (Contrib) `receivercreator`: Add support for starting logs' collection based on provided k8s annotations' hints ([#34427](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/34427))
+- (Contrib) `ottl`: Add a new ottl trim function that trims leading and trailing characters from a string (default- whitespace). ([#34100](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/34100))
+- (Contrib) `sqlqueryreceiver`: Add instrumentation scope to SQL query receiver metrics and logs ([#31028](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/31028))
+- (Contrib) `statsdreceiver`: Add UDS support to statsdreceiver ([#21385](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/21385))
+- (Contrib) `tailsamplingprocessor`: Support hot sampling policy loading ([#37014](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/37014))
+- (Contrib) `cmd/telemetrygen`: Introduce support for generating histograms in telemetrygen ([#36322](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36322))
```

Comment:
Do we generally add telemetrygen changes?

Reply:
No. We don't ship telemetrygen